### PR TITLE
City administration 3

### PR DESCRIPTION
**Pull request by IvanSultanov**
Added requests confirmation from sports, implemented WebApi version of ResidentAccounts CRUD, pagination, live clock, search...

### CODE REVIEW

File: UtopiaCity/wwwroot/js/CityAdministration/Edit.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+﻿window.onload = function () {
+    var id = window.location.search?.substring(4);
```
Comment (crocomoth) [inline UtopiaCity/wwwroot/js/CityAdministration/Edit.js:2]: looks scary

File: UtopiaCity/wwwroot/js/CityAdministration/Edit.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    const gender = form.elements["gender"].value == "0" ? 0 : 1;
+    const marriageId = form.elements["marriageId"].value == "" ? null : form.elements["marriageId"].value;
+    const birthPlace = form.elements["birthPlace"].value;
+    const registrationAddress = form.elements["registrationAddress"].value;
+    const medicalRecords = form.elements["medicalRecords"].value;
+    const property = form.elements["property"].value;
+    const motorTransport = form.elements["motorTransport"].value;
+    const email = form.elements["email"].value;
+    const phone = form.elements["phone"].value;
+    if (id == 0)
```
Comment (crocomoth) [inline UtopiaCity/wwwroot/js/CityAdministration/Edit.js:80]: maybe it makes sense to make an object of some sort? Yes you have parsed the response, "validated" it, but 10+ parameters is too much. If only JS had strongly typed types :)

**Review by crocomoth — APPROVED** [review]

File: UtopiaCity/wwwroot/js/CityAdministration/Edit.js (hunk 1 above)
Comment (IvanSultanov) [reply UtopiaCity/wwwroot/js/CityAdministration/Edit.js:2]: Maybe there are some better solutions to pass Id from one page to another )

File: UtopiaCity/wwwroot/js/CityAdministration/Edit.js (hunk 2 above)
Comment (IvanSultanov) [reply UtopiaCity/wwwroot/js/CityAdministration/Edit.js:80]: Yes, definitely it should be done another way. I will look for a solution.

File: UtopiaCity/wwwroot/js/CityAdministration/Edit.js (hunk 1 above)
Comment (IvanSultanov) [reply UtopiaCity/wwwroot/js/CityAdministration/Edit.js:2]: Could you please advice something here?

File: UtopiaCity/wwwroot/js/CityAdministration/Edit.js (hunk 2 above)
Comment (IvanSultanov) [reply UtopiaCity/wwwroot/js/CityAdministration/Edit.js:80]: Refactored methods with too many parameters, could you please check it?

**Review by Ibraga96 — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Hi

**Pull request merged**